src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -103,6 +78,31 @@ boolean verifyUsername(String usernameToVerify) {
     return matcher.matches();
   }
 
+  /**
+   * Prompts the user for a valid password. Password must not be empty or include spaces.
+   *
+   * @return validated password.
+   */
+  String getPassword() {
+    out.println("Enter your password:");
+    password = scanner.nextLine();
+    while (!verifyPassword(password)) {
+      err.println("You did not enter a password.\n" + "Please, enter your password:");
+      password = scanner.nextLine();
+    }
+    return password;
+  }
+
+  /**
+   * Verifies the password isn't empty and doesn't contain spaces.
+   *
+   * @param passwordToVerify is the string to be assessed.
+   * @return <code>true</code> if the password entered is valid; <code>false</code> otherwise.
+   */
+  private boolean verifyPassword(String passwordToVerify) {
+    return !passwordToVerify.isEmpty() && !passwordToVerify.contains(" ");
+  }
+
   /**
    * Prompts the user for a valid host name, which follows the below parameters: (1) Must be
    * alphanumeric (2) Can't be longer than 255 characters (3) Host name segments cannot exceed 63
```

Comment:
```suggestion
   * Prompts the user for a valid hostname, which follows the below parameters: is alphanumeric, shorter than 255 characters, with name segments not exceeding 63 characters, and starts and ends with alphanumeric characters.
```
Using the same wording as the `verifyHostname()` method